.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.191.1/containers/javascript-node/.devcontainer/base.Dockerfile
+
+# [Choice] Node.js version: 16, 14, 12
+ARG VARIANT="16-buster"
```

Comment:
Here it's `16` but in `decontainer.json` it's `12`. Is that intended?

Reply:
That's intended. "16-buster" is the default value to ensure that the docker file is also working standalone, and we overwrite the variant arg to "12" in the `devcontainer.json`.